cmd/hub/cli/version.go

```diff
@@ -0,0 +1,91 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/logrusorgru/aurora"
+	su "github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/spf13/cobra"
+	bi "github.com/textileio/textile/buildinfo"
+	"github.com/textileio/textile/cmd"
+)
+
+func checkProduction() (*su.Release, error) {
+	config := su.Config{
+		Filters: []string{
+			"hub",
+		},
+	}
+	updater, err := su.NewUpdater(config)
+	if err != nil {
+		return nil, err
+	}
+
+	latest, found, err := updater.DetectLatest(cmd.Repo)
+	if err != nil || !found {
+		return nil, err
+	}
+	return latest, nil
+}
+
+func getAPIVersion() (string, error) {
+	ctx, cancel := context.WithTimeout(Auth(context.Background()), cmd.Timeout)
+	defer cancel()
+	res, err := clients.Hub.BuildInfo(ctx)
+	if err != nil {
+		return "", err
+	}
+	if res.Version == "development" {
+		return fmt.Sprintf("%s (%s)", res.GitCommit, res.BuildDate), nil
+	}
+	return res.Version, nil
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Show current version",
+	Long:  `Shows the installed CLI version.`,
+	Args:  cobra.ExactArgs(0),
+	Run: func(c *cobra.Command, args []string) {
+		version := bi.Version
+
+		apiVersion, err := getAPIVersion()
```

Comment:
this will also let you know what the API you are connected to is running. default `development` if you are building with docker-compose-dev, but again, if that's the case it will print the commit info running on the api